spec/validators/claim/base_claim_sub_model_validator_spec.rb

```diff
@@ -4,6 +4,10 @@
   let(:claim) { FactoryBot.create :claim }
   let(:defendant) { claim.defendants.first }
 
+  it_behaves_like 'a custom CCCD associated error handler' do
+    subject(:validator) { described_class.new }
```

Comment:
Does this need to be a `subject`? I don't think it is being used as the subject of any tests. Likewise for the other places where the new shared examples are used.

Not a major issue. Maybe it is clearer like this.

Reply:
have added [this commit](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/4080/commits/db6be30299f56577ee458f158b4c7b0a315b56e4)  which makes it more consistent and places the subject in its "proper" context  - i.e. the class under test.